### ISSUE TITLE
One of several sensors added in conf file not listed as entity 

### ISSUE THREAD

**Issue by lukaszliniewicz** [custom-components/ble_monitor#236]
I'm currently testing a setup with several xiaomi temp sensors, and it works very well, with just one problem - one of the six sensors I defined in configuration.yaml is not displayed in HA. Is this because it is simply not detected (out of range) or may something else be wrong? Others show up even when "unavailable" and grayed-out. Apologies if this is just a matter of BT range. 
![HA entities](https://user-images.githubusercontent.com/75737665/104636404-9ab2d000-56a3-11eb-921a-d84292c1be88.PNG)
![conf](https://user-images.githubusercontent.com/75737665/104636406-9be3fd00-56a3-11eb-8edc-f820a4d5c952.PNG)

**Comment by Ernst79** [comment]
Let me describe how is works. If your sensor is added to devices (with `mac: xxxx`) , it always show up. The first period (60 seconds in your case) it will be `unknown` or `unavailable` as it is collecting all messages in this period and performing the rounding to 1 decimal. If it receives at least one message from that mac address, it should show the value. Normally, if it doesn't show up, its a matter of bad reception. Unfortunately, Bluetooth reception can be affected by many things, most common are SSD drives and wifi signals. The FAQ gives some examples and solutions (external dongles, extention cables). Some sensors are also slower than others. Which sensors do you have?

**Comment by estebanbouza** [comment]
Having the exact same issue here. Devices are added via configuration.yaml the same as the screenshot before. 
Some of them show the actual values, some of them are out of range and show "unavailable". This is understandable.

However, if I add a new device, the new device is nowhere to be seen in the HA ui (not in devices, sensors, entities). If I click on Configuration / Integrations, the tile for Passive BLE Monitor shows e.g. 2 devices when I have added 3 in the yaml array. 
I have performed server reboots and configuration reloads, but it doesn't seem to affect it. 

Could this be a bug or am I missing something?
